### ISSUE TITLE
BLE and WiFi support in MCU SDK 2.0?

### ISSUE THREAD

**Issue by kaofishy** [bouffalolab/bouffalo_sdk#46]
Hi there,

Are there any plans to support BLE and WiFi like version 1.4.5 and earlier did? I'm specifically interested in both BL702 and BL808. I see that BL808 is only supported in 2.0, so is there no official way to develop BLE/WiFi firmware for BL808?

Thanks,
Tony

**Comment by gamelaster** [comment]
AFAIK MCU SDK supported only BLE.

**Comment by kaofishy** [comment]
> AFAIK MCU SDK supported only BLE. 

You're right, it was BLE only. The issue is that BLE is missing from the "new" MCU SDK.

**Comment by gamelaster** [comment]
@kaofishy correct. MCU SDK seems to be only for MCU usage, so even I was surprised to see BLE support. I think RF protocols will be supported only in IOT sdk

**Comment by kaofishy** [comment]
If that is truly the case then it just seems like unnecessary segmentation, it's not like the IoT SDK is under an NDA or anything, just adding confusion.

**Comment by sakumisu** [comment]
Please wait! All the wireless protocols will support. It is in refactoring.

**Comment by sakumisu** [comment]
You can get some secrets in [CMakeLists](https://github.com/bouffalolab/bl_mcu_sdk/blob/master/CMakeLists.txt)

**Comment by kaofishy** [comment]
@sakumisu oh that's great to hear, thanks for the quick reply and keep up the good work, the new API is much nicer to work with than the old one.

**Comment by kamushadenes** [comment]
Hello folks, any ETA on the WiFi support?

**Comment by sakumisue** [comment]
done

**Comment by andrewleech** [comment]
Does this mean wifi and/or ble support is now available on BL808?
If so could you please link to some details?

**Comment by sakumisu** [comment]
> Does this mean wifi and/or ble support is now available on BL808? If so could you please link to some details?

Readme has noted, only for bl616/bl618

**Comment by andrewleech** [comment]
I saw that and was hoping it was out of date... this issue was raised for bl702 and bl808, if their support is still pending the issue should be left open!

**Comment by Fishwaldo** [comment]
Hey @sakumisu is there still plans to release wifi for the BL808?

**Comment by sfranzyshen** [comment]
@sakumisu why can't the bl808 community ever get any answers about BLE and WiFi support ON THE BL808 platform???

Will wifi/ble  "EVER" be an option for the bl808 platform ??? ... 

I am not asking "WHEN" ... I am asking "IF EVER" ???

can you at least answer that??

**Comment by gamelaster** [comment]
@sfranzyshen there is wifi support on the BL808 platform. Bouffalo provided blobs both for Linux (those are fresh ones), and then ones are in IoT SDK.

**Comment by sfranzyshen** [comment]
> @sfranzyshen there is wifi support on the BL808 platform. Bouffalo provided blobs both for Linux (those are fresh ones), and then ones are in IoT SDK.

I am asking about HERE.

**Comment by sfranzyshen** [comment]
still nothing ... what a shame
